netgrasp/netgrasp.py

```diff
@@ -904,7 +903,6 @@ def device_request(ip, mac):
         # Log request.
         ng.db.cursor.execute("SELECT request.rid, request.active FROM request WHERE request.ip = ? ORDER BY updated DESC LIMIT 1", (ip,))
         seen = ng.db.cursor.fetchone()
-        rid, active = (False, False)
```

Comment:
I read through the code paths/thought about that one for quite a bit. After going back and forth I pulled it out thinking overall it was better not being there.